apps/dcos_dns/src/dcos_dns.erl

```diff
@@ -170,31 +171,68 @@ push_zone(ZoneName, Records) ->
 -spec(push_prepared_zone(dns:dname(), [dns:dns_rr()]) ->
     ok | {error, Reason :: term()}).
 push_prepared_zone(ZoneName, Records) ->
+    Begin = erlang:monotonic_time(),
     try erldns_zone_cache:get_zone_with_records(ZoneName) of
         {ok, #zone{records=Records}} ->
             ok;
         _Other ->
-            Hash = crypto:hash(sha, term_to_binary(Records)),
-            case erldns_zone_cache:put_zone({ZoneName, Hash, Records}) of
-                ok ->
-                    lager:notice(
-                        "DNS Zone ~s was updated (~p records, sha: ~s)",
-                        [ZoneName, length(Records), bin_to_hex(Hash)]),
-                    ok;
-                {error, Error} ->
-                    lager:error(
-                        "Failed to push DNS Zone \"~s\": ~p",
-                        [ZoneName, Error]),
-                    {error, Error}
-            end
+            put_zone(Begin, ZoneName, Records)
     catch error:Error ->
         lager:error(
-            "Failed to push DNS Zone \"~s\": ~p",
-            [ZoneName, Error]),
+            "Failed to push DNS Zone \"~s\": ~p", [ZoneName, Error]),
```

Comment:
nit: for consistency, could you please use one code style? either of them is fine

https://github.com/dcos/dcos-net/pull/127/files#diff-19215963023a2b45a6e2cd51922592d6R217

Reply:
Yeah. I also just noticed that I broke the lines inconsistently on the Prometheus metrics call. 

Is it possible that we codify those on a linter? Coming from JS, I am really used to just have this being done automatically for me by a tool...

Reply:
Unfortunately, [elvis](https://github.com/inaka/elvis_core/blob/master/src/elvis_style.erl) doesn't support such things.